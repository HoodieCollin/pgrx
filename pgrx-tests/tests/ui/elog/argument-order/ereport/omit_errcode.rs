use pgrx::prelude::*;

fn main() {}

#[pg_extern]
fn ereport_errcode_is_required() {
    ereport!(
        loglevel = PgLogLevel::LOG;
        message = "...";
    );
}
