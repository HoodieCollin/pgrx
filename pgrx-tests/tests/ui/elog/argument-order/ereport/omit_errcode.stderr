error: The alternative invocation style requires `errcode` argument to be specified after `loglevel`.
  --> tests/ui/elog/argument-order/ereport/omit_errcode.rs:7:5
   |
7  | /     ereport!(
8  | |         loglevel = PgLogLevel::LOG;
9  | |         message = "...";
10 | |     );
   | |_____^
   |
   = note: this error originates in the macro `ereport` (in Nightly builds, run with -Z macro-backtrace for more info)
